src/main/java/frc/robot/subsystems/flywheel/FlyWheelIONonVariable.java

```diff
@@ -24,7 +24,7 @@ public FlyWheelIONonVariable() {
                 this.topFlyWheel = new WPI_TalonFX(10);
                 break;
             case ROBOT_2022_PRACTICE:
-                break;
+                
```

Comment:
```suggestion
```

Reply:
We are removing break because it will cascade to the next case, which happens to be `default`.